libraries/botbuilder-dialogs-adaptive/src/actions/httpRequest.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * @module botbuilder-planning
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { DialogTurnResult, DialogConfiguration, DialogContext, DialogCommand, Dialog } from 'botbuilder-dialogs';
+import { Activity } from 'botbuilder-core';
+import { ExpressionProperty, ExpressionPropertyValue } from '../expressionProperty';
+import * as rp from 'request-promise-native';
+
```

Comment:
we should be using node-fetch everywhere as it can be easily polyfilled to run on a browser